.github/workflows/ci.yml

```diff
@@ -79,18 +81,30 @@ jobs:
           - scenario: ember-release
             browser: Firefox
             bootstrap: 5
-          - scenario: node-tests
           - scenario: tracked-toolbox-v1
             bootstrap: 5
     steps:
       - name: Checkout code
         uses: wyvox/action@v1
       - name: Test
+        working-directory: test-app
         run: pnpm ember try:one ${{ matrix.scenario }}
         env:
           BOOTSTRAPVERSION: ${{ matrix.bootstrap }}
           BROWSER: ${{ matrix.browser }}
 
+  test-node:
```

Comment:
Great if this works! But just to mention: when migrating to v2, these tests can be dropped (as the feature they test will be dropped as well), so probably not worth spending much time on these, should they make any trouble now...

Reply:
Ah, that would be nice. It feels a bit dirty to have it in the addon directory, but it's the only way that it will work